assets/dev/js/editor/controls/media.js

```diff
@@ -95,7 +120,7 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 			states: [
 				new wp.media.controller.Library( {
 					title: __( 'Insert Media', 'elementor' ),
-					library: wp.media.query( { type: this.getMediaType() } ),
+					library: wp.media.query( { type: this.getLibraryType( mediaType ) } ),
```

Comment:
```suggestion
					library: wp.media.query( { type: this.currentLibraryType } ),
```

Reply:
@StyleShit - Please also change below (it was my mistake) - line 134:
From:
this.setUploadMimeType( this.frame, this.getMediaType() );
To:
this.setUploadMimeType( this.frame, mediaType );